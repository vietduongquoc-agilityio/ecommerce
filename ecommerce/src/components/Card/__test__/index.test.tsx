import { render } from "@testing-library/react";
import ItemCard from "../";

describe("ItemCard Component", () => {
  it("matches snapshot", () => {
    const { asFragment } = render(<ItemCard />);
    expect(asFragment()).toMatchSnapshot();
  });
});
