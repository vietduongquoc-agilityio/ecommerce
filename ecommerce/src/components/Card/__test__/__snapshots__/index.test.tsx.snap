// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ItemCard Component matches snapshot 1`] = `
<DocumentFragment>
  <article
    style="position: relative; width: 285px; height: 446px; background: rgb(244, 245, 247); overflow: hidden; cursor: pointer;"
  >
    <img
      alt="leviosa"
      data-nimg="1"
      decoding="async"
      height="40"
      loading="lazy"
      src="/_next/image?url=%2Fimg.jpg&w=96&q=75"
      srcset="/_next/image?url=%2Fimg.jpg&w=48&q=75 1x, /_next/image?url=%2Fimg.jpg&w=96&q=75 2x"
      style="color: transparent; width: 100%; height: 300px; object-fit: cover;"
      width="40"
    />
    <div
      style="padding: 16px;"
    >
      <p
        style="color: rgb(58, 58, 58); font-weight: 600; font-size: 24px; margin-bottom: 8px;"
      >
        Syltherine
      </p>
      <p
        style="color: rgb(159, 159, 159); font-weight: 500; font-size: 16px; margin-bottom: 8px;"
      >
        Stylish cafe chair
      </p>
      <p
        style="color: rgb(58, 58, 58); font-weight: 600; font-size: 20px;"
      >
        Rp 2.500.000
      </p>
    </div>
    <div
      style="position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; background-color: rgb(58, 58, 58); opacity: 0; display: flex; align-items: center; justify-content: center; transition: opacity 0.3s ease;"
    >
      <button
        style="padding: 12px 52px; font-size: 18px; font-weight: 600; text-align: center; width: auto; cursor: pointer; transition: all 0.3s ease; background-color: rgb(255, 255, 255); color: rgb(184, 142, 47);"
      >
        Add to Cart
      </button>
    </div>
  </article>
</DocumentFragment>
`;
